packages/webapp/src/containers/AddFarm/index.js

```diff
@@ -253,6 +245,15 @@ const AddFarm = () => {
     navigator.geolocation.getCurrentPosition(handleGetGeoSuccess, handleGetGeoError, getGeoOptions);
   };
 
+  //TODO: update hookform, move all states in hookform, and rewrite the component
+  useEffect(() => {
+    if (farmAddress) {
+      setTimeout(() => {
```

Comment:
`setTimeout` may improve the UI behavior, but I think that there is probably a deeper and better fix by reworking some of the container. I'm working on that.